ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1060,3 +1060,33 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-for-pull-request':    
+           test_name: stop-all-stale-jobs
+           node: 'antrea-stop-all'
+           description: 'This is the {test_name} test for {name}.'
+           branches:
+           - ${{sha1}}
+           builders:
+             - builder-stop-all:
+               jenkins_location: lab
```

Comment:
I didn't find where this jenkins-job-builder variable is used. Can you point out for me?

Reply:
Hi, it was an unused variable I removed that.